### PR TITLE
Publish December 2017 issue of QAT Gazette

### PR DESCRIPTION
**Pull request by JBHyperion**
### Description

Add December issue of QAT Gazette blog

### Status

Finished, pending review

### Changelog

2017/12/12 - Initial PR

### CODE REVIEW

File: news/2017-12-13-qat-gazette-december-2017.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+layout: post
+title: "The QAT Gazette: December 2017"
+date: 2017-12-13 09:00:00 +0000
+---
+
+An update to the Beatmap Nominator applications process and more clarity on the Beatmap Nominator probation phase, adjustments to beatmap Hype, as well as how to apply and contribute as a Metadata Team Helper - all in store for this month’s issue of the QAT Gazette!
```
Comment (MegaApplePi) [inline news/2017-12-13-qat-gazette-december-2017.md]: `’` -> `'`
(and repeat for other occurrences)

File: news/2017-12-13-qat-gazette-december-2017.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+Anyone is welcome to join the [Metadata Heap server](https://discord.gg/9Y4EdyM) if the above seems useful to you, but we’re especially interested in recruiting some dedicated and knowledgeable Helpers to assist our current team of experts. More details can be found on the server’s #announcements channel, but if you’re already involved there, head straight on over to the [application form](https://goo.gl/87mEQA) and show your interest!
+
+### Modifications to the beatmap Hype system
+
+Anyone who has participated in the Modding Discussions system will have noticed the “Hype Train” feature - billed as a replacement for kudosu star shooting from the previous forum modding system. Since it’s not tied to kudosu, however, Hype offers an opportunity for players to get involved in the promotion of beatmaps, without the need to necessarily mod them.
```
Comment (MegaApplePi) [inline news/2017-12-13-qat-gazette-december-2017.md]: `“` and `”` -> `"`
(this also occurrences for the paragraph below)

File: news/2017-12-13-qat-gazette-december-2017.md [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+
+* Users will be required to play through at least one difficulty before being able to Hype a mapset.
+* Users will be limited to a certain (as yet undecided) number of Hype per month, making each one feel more valuable
+* Total number of Hype required for a beatmap to be nominated for Ranking will be reduced to 5 (to account for the increased worth of each).
+
+The details of these modifications are subject to change based on community reception, of course. Be sure to give them a try and leave feedback for future development!
+
+### Quality Assurance Team changes
+
+To coincide with the recent Beatmap Nominator promotions, I am pleased to welcome the **newest member of the Quality Assurance Team**, [Protastic101](https://osu.ppy.sh/u/6712747)! After a year of impressive growth in the Beatmap Nominator team and with maturity beyond her years, she will be soon be assisting with the checking of Qualified maps and assisting new Beatmap Nominators in osu!mania. Please offer her your congratulations and wish her the best of luck with her new responsibilities!
```
Comment (MegaApplePi) [inline news/2017-12-13-qat-gazette-december-2017.md]: 🤷‍♂️ remain consistent with linking?
`https://osu.ppy.sh/u/6712747` -> `https://osu.ppy.sh/users/6712747`

File: news/2017-12-13-qat-gazette-december-2017.md [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+date: 2017-12-13 09:00:00 +0000
+---
+
+An update to the Beatmap Nominator applications process and more clarity on the Beatmap Nominator probation phase, adjustments to beatmap Hype, as well as how to apply and contribute as a Metadata Team Helper - all in store for this month’s issue of the QAT Gazette!
+
+![](/wiki/shared/news/banners/qat-blog-logo.png)
+
+Seasons Greetings!
+
+I mentioned last month that I was looking forward to the progression and development of the **newly introduced Modding Discussions system**, and thanks to numerous contributions from our front-line Beatmap Nominators and modders, we've seen a number of quality of life improvements. These include the addition of line breaks in suggestions, more visibility for pending discussions per difficulty, and perhaps most importantly, the ability for Beatmap Nominators to counter (or "veto") a nomination by posting a Problem stamp.
```
Comment (MegaApplePi) [inline news/2017-12-13-qat-gazette-december-2017.md]: according to @osuShiro, it's officially called "Beatmap Discussion"... but whatever suits your fancy ¯\\\_(ツ)\_/¯

File: news/2017-12-13-qat-gazette-december-2017.md (hunk 1 above)
Comment (JBHyperion) [reply news/2017-12-13-qat-gazette-december-2017.md]: Not quite sure how I achieved this lol, but fixed all apostrophes

File: news/2017-12-13-qat-gazette-december-2017.md (hunk 4 above)
Comment (JBHyperion) [reply news/2017-12-13-qat-gazette-december-2017.md]: Fair enough, better to be consistent I guess

File: news/2017-12-13-qat-gazette-december-2017.md (hunk 2 above)
Comment (JBHyperion) [reply news/2017-12-13-qat-gazette-december-2017.md]: Once again, not sure why these morphed into the curlier variety, but fixed

File: news/2017-12-13-qat-gazette-december-2017.md (hunk 3 above)
Comment (JBHyperion) [reply news/2017-12-13-qat-gazette-december-2017.md]: Whoops, fixed

**Pull request merged**